src/util/amount.rs

```diff
@@ -1375,4 +1423,22 @@ mod tests {
         let value_without: serde_json::Value = serde_json::from_str("{}").unwrap();
         assert_eq!(without, serde_json::from_value(value_without).unwrap());
     }
+
+    #[cfg(feature = "serde")]
+    #[test]
+    fn serde_parse() {
+        use serde_json;
+
+        #[derive(Deserialize, PartialEq, Debug)]
+        struct T {
+            #[serde(default, deserialize_with = "::util::amount::serde::parse")]
+            pub amt: Amount,
+            #[serde(default, deserialize_with = "::util::amount::serde::parse")]
+            pub samt: SignedAmount,
+        }
+
+        let t: T = serde_json::from_str("{\"amt\": \"2 BTC\", \"samt\": \"-5 sat\"}").unwrap();
```

Comment:
JFYI, Nit: this can be done w/o quote escapes with raw string literals
```suggestion
        let t: T = serde_json::from_str(r#"{"amt": "2 BTC", "samt": "-5 sat"}"#).unwrap();
```